modules/mobile/spec/request/appointments_vaos_v2_list_request_spec.rb

```diff
@@ -217,7 +217,7 @@
           end
         end
         expect(response).to have_http_status(:ok)
-        expect(JSON.parse(response.body)['data'].size).to eq(1305)
+        expect(JSON.parse(response.body)['data'].size).to eq(980)
```

Comment:
If I had to guess, I'd speculate the missing 325 results are the result of either appointment requests with the "booked" status or perhaps appointments with missing dates. Something along those lines.